### ISSUE TITLE
I can not push 8g model to Ollama

### ISSUE THREAD

**Issue by hololeo** [ollama/ollama#6466]
### What is the issue?

using 

>ollama push me/mymodel

im getting upload rate like this

> 39 MB/8.5 GB  1.4 MB/s   1h41m

it will chug chug chug to 40 or 50% then i get the error

> Error: max retries exceeded: http status 502 Bad Gateway: <?xml version="1.0" encoding="UTF-8"?><Error><Code>InternalError</Code><Message>We encountered an internal connectivity issue. Please try again.</Message></Error>

im wondering if my ISP is getting me off? Or perhaps im not uploading fast enough so ollama server cuts me off?

any tips on how to push largish models? it will do 4g ok

### OS

macOS

### GPU

Apple

### CPU

Apple

### Ollama version

0.3.6

**Comment by igorschlum** [comment]
Hi @hololeo can you use a website like fast.com to share your download or upload speed? It will help.

**Comment by hololeo** [comment]
![Screenshot 2024-08-23 at 8 20 53 AM](https://github.com/user-attachments/assets/44c53546-d621-4845-9051-eaf4cfb09cba)

**Comment by hololeo** [comment]
bump? any indication of what I can do on my end to fix this? I suppose moving to better internet but thats difficult and expensive in my locale

**Comment by igorschlum** [comment]
Hi @hololeo, did you try using version 0.3.9? The number of parallel downloads has been adjusted, so it might work better now.

**Comment by jmorganca** [comment]
@hololeo as @igorschlum mentioned does the latest version happen to work? If not we will continue improving the networking code to fix this soon – sorry about the issue.

**Comment by hololeo** [comment]
yes i have ollama version 0.3.9. The error still occurs. :(. Thank you for your work and  investigating

**Comment by igorschlum** [comment]
@hololeo please can you share a screen with the upload ans download speed you get using fast.com?

**Comment by purin-blog** [comment]
PS C:\Users\Admin\.ollama\models> ollama create -f Modelfile white_/mario
transferring model data 
using existing layer sha256:8de95da68dc485c0889c205384c24642f83ca18d089559c977ffc6a3972a71a8 
using existing layer sha256:62fbfd9ed093d6e5ac83190c86eec5369317919f4b149598d2dbb38900e9faef 
using existing layer sha256:c156170b718ec29139d3653d40ed1986fd92fb7e0959b5c71f3c48f62e6636f4 
creating new layer sha256:b3fb3caebac98f848f5c65ff9eb77c6f3ce8cefd4fad45e94c37725ca847c562 
creating new layer sha256:b1c932e03beb32c4ab61bb50b2fa06ab1f2ea4e99ee6495670bbe23834dc7d62
creating new layer sha256:ed36a163da40b1fea25ee506c2b82eb76df4878b4e2afa5db53f42f7a97639e5
writing manifest
success
PS C:\Users\Admin\.ollama\models> ollama push white_/mario              
retrieving manifest
Error: file does not exist
PS C:\Users\Admin\.ollama\models>

**Comment by purin-blog** [comment]
Always fail

**Comment by hololeo** [comment]
still not possible
